### ISSUE TITLE
Packet Parser: Unknown Switch: 0AB8: IRO Restart

### ISSUE THREAD

**Issue by legendbear9** [OpenKore/openkore#1293]
Hi everyone! 
1. I got this text on my openkore when a status skill (Impositus, Increase Agi, Bless) is no longer. I read from main page that this could become indirect evidence. Indeed, one of my account blocked due to this :( Can sb help me to fix this ?
2. Besides I tried to not put Blue Gemstone 0 0 0  or some other items to storage but failed to do this. Some numbers are put after spells but I could not find it. Please help.
Thanks all

**Comment by Youtouchmytralala** [comment]
http://forums.openkore.com/

**Comment by fadreus** [comment]
Packet 0AB8 is due to skills or items is applying buff or debuff.
It has nothing to do with your account being blocked (unless you are doing it too quickly and server detects unusual behavior from your account which indirectly related to anti-bot feature warpportal implemented).

> Besides I tried to not put Blue Gemstone 0 0 0 or some other items to storage but failed to do this.

I believe solution for this can be found on other issue since I read it somewhere (I'm not playing RE:Start so I didn't pay much attention on the comments)

> Some numbers are put after spells but I could not find it. Please help.

Spells? Magic Spells ID or the spelling in items_control.txt?
If you referring to items ID, on general you can refer items ID at http://ratemyserver.net or look inside tables/iro folder.

**Comment by fadreus** [comment]
![image](https://user-images.githubusercontent.com/32188494/31838863-22358892-b611-11e7-9e06-3395c0e2b736.png)

![image](https://user-images.githubusercontent.com/32188494/31838880-30e2e11e-b611-11e7-98ed-1a2658b1ff23.png)

**Comment by legendbear9** [comment]
Thank you for the ID item number, I forgot web ratemyserver.net because it doesn't fit well to Iro Restart. 
I will join 4rom to read more. :D 
Related to 1st issue,  I mean how to fix Packet 0AB8 or just let it be ?

**Comment by fadreus** [comment]
Just let it be. It didn't harm you (Except the mini-heart attack of yellow word printing)🤣

Dev team have no idea too what this packet do but I only see this packets when ever it tried to cast skills before another one is completed or the buff / debuff skills attempts to inflict the buff/debuff quickly and failed (frost nova, meteor storm, etc etc). I tried with non-buff / debuff skills, and the error didn't appear (probably my character isn't just fast enough to trigger this? IDK).

Dev probably think the same way I did but if there is no fix from them, means this probably not the real issue.
